coalib/results/Diff.py

```diff
@@ -166,6 +168,9 @@ def from_clang_fixit(cls, fixit, file):
         """
         assert isinstance(file, (list, tuple))
 
+        if get_distribution('libclang-py3').version != '3.4.0':
+            raise LibclangError('coala requires clang 3.4')
```

Comment:
I would like to see this at module scope, this may be a significant performance hit for LocalBears using this^^